### ISSUE TITLE
Sarnax is a scammer

### ISSUE THREAD

**Issue by r2000t** [Customscript/Sarnax-7522-is-a-SCAMMER-BEWARE#1]
Sarnax is scammer and still working here in github and not getting banned idk wtf is going on... he scammed me too .. he scammed so many member and i saw some of them here telling this and no one care .. wtf ?! pls bann him from github ... 
https://github.com/Customscript/Sarnax-7522-is-a-SCAMMER-BEWARE

**Comment by Customscript** [comment]
Yeah he’s doing the same thing over and over to many people. Notice on GitHub, his Repo are set without Issues included. So, that no one can comment on his scams.

**Comment by r2000t** [comment]
I found Sarnax have a new name on github he still doing this here the name is : Fnoberz 
he start doing this again ...

**Comment by DeletedUser0981** [comment]
He just scammed me too for 180 dollars crypto he claims to be making cheats but blocks and bans what a stupid retard, i should have searched him up before buying from him FUCK!

**Comment by bboychris168** [comment]
Yeah shit i got scammed as well. This guy has to be banned from github. Its crazy how long hes been doing this. I also should have done my research before buying off him........

**Comment by r2000t** [comment]
New name of him is " Slackes " idk why git hub still let him doing this shit

**Comment by devk14** [comment]
His new name is "Slackes" be careful
